ale_linters/ruby/brakeman.vim

```diff
@@ -1,9 +1,17 @@
 " Author: Eddie Lebow https://github.com/elebow
 " Description: Brakeman, a static analyzer for Rails security
 
+call ale#Set('ruby_brakeman_options', '')
+call ale#Set('ruby_brakeman_executable', 'brakeman')
+
 let g:ale_ruby_brakeman_options =
 \   get(g:, 'ale_ruby_brakeman_options', '')
 
+function! ale_linters#ruby#brakeman#GetExecutable(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'ruby_brakeman_executable')
+    return ale#handlers#ruby#EscapeExecutable(l:executable, 'brakeman')
```

Comment:
The answer here is that you should just return the executable here wihout escaping it, and you should use `ale#handlers#ruby#EscapeExecutable` in `GetCommand` below. The reason being that `executable_callback` should return the path to the program that will be run, and `command_callback` should return a command to run, where the part of the command with an executable name has to be escaped.

Reply:
Ok, cool. That is what I was thinking, but I wanted to check with you first.